src/elm/Pages/Build.elm

```diff
@@ -355,22 +371,32 @@ viewRecentBuild now timezone org repo buildNumber build idx =
                 class ""
     in
     a
-        [ class "recent-build"
+        [ class "recent-build-link"
+        , Util.testAttribute <| "recent-build-link-" ++ String.fromInt buildNumber
         , currentBuildClass
         , Routes.href <| Routes.Build org repo (String.fromInt build.number) Nothing
         , attribute "aria-label" <| "go to previous build number " ++ String.fromInt build.number
         ]
         [ icon
-        , div [ class "tooltip", Util.testAttribute "build-history-tooltip" ]
-            [ div [ class "-info" ]
-                [ div [ class "-line", class "-header" ]
-                    [ span [ class "-number" ] [ text <| String.fromInt build.number ]
-                    , span [ class "-event" ] [ text build.event ]
-                    ]
-                , div [ class "-line" ] [ span [ class "-label" ] [ text "started:" ], span [ class "-content" ] [ text <| Util.dateToHumanReadable timezone build.started ] ]
-                , div [ class "-line" ] [ span [ class "-label" ] [ text "finished:" ], span [ class "-content" ] [ text <| Util.dateToHumanReadable timezone build.finished ] ]
-                , div [ class "-line" ] [ span [ class "-label" ] [ text "duration:" ], span [ class "-content" ] [ text <| Util.formatRunTime now build.started build.finished ] ]
+        ]
+
+
+{-| recentBuildTooltip : takes time info and build and renders tooltip for viewing recent build info
+
+    tooltip is visible when the recent build link is focused or hovered
+
+-}
+recentBuildTooltip : Posix -> Zone -> Build -> Html msg
+recentBuildTooltip now timezone build =
+    div [ class "recent-build-tooltip", Util.testAttribute "build-history-tooltip" ]
+        [ div [ class "info" ]
+            [ div [ class "line", class "header" ]
+                [ span [ class "number" ] [ text <| String.fromInt build.number ]
+                , span [ class "event" ] [ text build.event ]
```

Comment:
i see `.event` just applies italic styling. as you want to stress this keyword visually, why not use `<em>` to call it out.